react/features/mobile/navigation/components/HeaderNavigationButton.tsx

```diff
@@ -64,6 +69,7 @@ const HeaderNavigationButton = ({ disabled, label, onPress, src, style, twoActio
             {
                 src ? (
                     <IconButton
+                        color = { src && color }
```

Comment:
Why is src taken taken into account here?

Reply:
It's not mandatory. I can remove this. I wanted to be more specific about the use of IconButton props but there is no need to do that.